data/language/nl-NL.yml

```diff
@@ -2191,3 +2191,209 @@ strings:
+  2215: "Geen vrachttype geselecteerd"
+  2216: "{SMALLFONT}{COLOUR BLACK}Open a station window to filter by station"
+  2217: "{SMALLFONT}{COLOUR BLACK}Select a cargo type from the list of available cargo"
+  2218: "Constructie: Previous tab"
+  2219: "Constructie: Next tab"
+  2220: "Constructie: Previous track piece"
+  2221: "Constructie: Next track piece"
+  2222: "Constructie: Previous slope"
+  2223: "Constructie: Next slope"
+  2224: "Constructie: Build at current position"
+  2225: "Constructie: Remove at current position"
+  2226: "Constructie: Select position"
+  2227: "Trains reverse at signals"
+  2228: Autom. aflossen
+  2229: "{SMALLFONT}{COLOUR BLACK}Gebruikt elke dag fondsen waarover je beschikt om je lening af te betalen."
+  2230: "{COLOUR WINDOW_2}Jaar:"
+  2231: "{COLOUR WINDOW_2}Maand:"
+  2232: "{COLOUR WINDOW_2}Dag:"
+  2233: Stel datum in
+  2234: "{COLOUR BLACK}{UINT16 RAW}"
+  2235: "{COLOUR BLACK}{INT32}"
+  2236: "{COLOUR BLACK}{INT32}"
+  2237: "{DATE DMY}"
+  2238: "Spelsnelheid: Normaal"
+  2239: "Spelsnelheid: Sneller"
+  2240: "Spelsnelheid: Extra snel"
+  2241: "{COLOUR WINDOW_2}Capaciteit: {COLOUR BLACK}{STRINGID}"
+  2242: "{NEWLINE}{COLOUR WINDOW_2} Obsolete: {COLOUR BLACK}{UINT16 RAW}"
+  2243: "Vehicle is locked!"
+  2244: "Display locked vehicles"
+  2245: "{SMALLFONT}{COLOUR BLACK}When enabled, will display obsolete and proposed vehicles"
+  2246: "{NEWLINE}{COLOUR WINDOW_2} Proposed: {COLOUR BLACK}{UINT16 RAW}"
+  2247: "Allow building locked vehicles"
+  2248: "{SMALLFONT}{COLOUR BLACK}When enabled, will allow building obsolete and proposed vehicles"
+  2249: "Build Vehicles Window"
+  2250: "{SMALLFONT}{COLOUR BLACK}Enable to change terrain type instead of elevation"
+  2251: "Invert right mouse dragging"
+  2252: "{SMALLFONT}{COLOUR BLACK}When enabled, panning the main game view will have inverted movement"
+  2253: "Show cash popups"
+  2254: "{SMALLFONT}{COLOUR BLACK}Toggle visibility of cash popups from construction and vehicles on/off"
+  2255: "Start server"
+  2256: "Sluit server"
+  2257: "Verbinding verbreken"
+  2258: "{COLOUR WINDOW_2}Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s), {INT32} wagons"
+  2259: "{COLOUR WINDOW_2}Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s)"
+  2260: "{COLOUR WINDOW_2} Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s)"
+  2261: "{NEWLINE}{COLOUR WINDOW_2} Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s)"
+  2262: "{NEWLINE}Lengte:  {INT32_2DP} tegel(s)"
+  2263: "{COLOUR WINDOW_2}{UINT16}/{UINT16}"
+  2264: "Reuzenscreenshot"
+  2265: "{COLOUR WINDOW_2} Bestandsnaam: {COLOUR BLACK}{STRINGID}"
+  2266: "Scenario-opties"
+  2267: "Objectselectie"
+  2268: "Gameplaytweaks"
+  2269: "Eigenaarsvoorkeuren"
+  2270: "Autosave-voorkeuren"
+  2271: "Stadsuitbreiding uitschakelen"
+  2272: "{SMALLFONT}{COLOUR BLACK}Wanneer ingeschakeld kunnen steden zich niet vernieuwen of uitbreiden over tijd."
+  2273: "Complete scenario challenge"
+  2274: "Wis"
+  2275: "Error loading objects"
```

Comment:
Hier zitten nog wat onvertaalde stukjes tekst.

Reply:
De PR had nog draftstatus, haha. Inmiddels ook de laatste beetjes vertaald, als 't goed is :)

Reply:
Ik zie dat er in de oude strings ook een partij onvertaalde zit. Dat is wel iets meer dan ik voor ogen had met deze PR.